### DISCUSSION TITLE
Serenity OOTB CSS issue

### DISCUSSION THREAD

**Discussion by ad-council** [serenity-is/Serenity#6307] [Help] [answered]
Hi Guyz,

I'm having a problem with loading CSS files. 

I created a basic new solution, and OOTB it does not load the CSS files. Please see the screenshot.

![serenity_error](https://user-images.githubusercontent.com/101110287/157029639-b338b457-2b35-408e-85af-4a8655bb460a.PNG)

Appreciate your help.

**Comment by ad-council** [comment]
Is there a reason that no one responds to theme related issue, but they respond to other discussion?

Am I missing something?

**Comment by minhhungit** [comment]
which version you are using, I'm using version 5.2.3.0 and I don't see your issue
Maybe you will need to upgrade your extension

![image](https://user-images.githubusercontent.com/2279508/157171455-6e32f7c3-3f01-404d-856f-68ae6d11ccf5.png)

**Reply by ad-council** [reply]
I'm using 5.2.3.0. If you see the screenshot, their is a 404 error in the Console view for default CSS file.

**Reply by ad-council** [reply]
This is the error I'm getting in browser console for serenity.css

"Failed to load resource: the server responded with a status of 404 ()"

**Comment by VictorTomaili** [comment, marked as answer]
Check your bundle configuration and ensure to run the `sergen restore` command. Your issue is depending on your environment. You can try to create a new empty project from serene to check for everything is well. If there is no problem with the new project, you need to figure out what is different in your project.

**Comment by ad-council** [comment]
When I ran `sergen restore`, I received below error

`Error while initializing MSBuildLocator: System.InvalidOperationException: No instances of MSBuild could be detected.
Try calling RegisterInstance or RegisterMSBuildPath to manually register one.
   at Microsoft.Build.Locator.MSBuildLocator.RegisterDefaults()
   at Serenity.CodeGenerator.MSBuild.MSBuildProjectSystem..cctor() in C:\Sandbox\StartSharp\Serenity\src\Serenity.Net.CodeGenerator\BuildProjectSystem\MSBuildProjectSystem.cs:line 15`

I have below version SDK already installed

`6.0.200 [C:\Program Files\dotnet\sdk]`

**Reply by VictorTomaili** [reply]
https://github.com/serenity-is/Serenity/issues/6305

**Reply by ad-council** [reply]
I appreciate your help, but please bare with me, I'm new to these RAD framework.

I have reinstalled SDK 6.0.2. I can verify that MSBuild executable is available at `C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin`

Do I have to define this path in the serenity configuration somewhere? because it did not work after SDK resintallation!

Thanks

**Reply by VictorTomaili** [reply]
You may need to install the `.Net SDK 5` instead of the `6`
This issue is related to the MSBuildLocator probably the same reason https://github.com/microsoft/MSBuildLocator/issues/152

**Reply by ad-council** [reply]
That worked out... Many thanks
